ogusa/SS.py

```diff
@@ -162,7 +162,7 @@ def inner_loop(outer_loop_vars, p, client):
     r_hh = aggr.get_r_hh(r, r_gov, K, D)
     bq = household.get_bq(BQ, None, p, 'SS')
     tr = household.get_tr(TR, None, p, 'SS')
-    ubi = p.ubi_nom_SS / factor
+    ubi = p.ubi_nom_array[-1, :, :] / factor
```

Comment:
@jdebacker . Why the change from `p.ubi_nom_SS` to `p.ubi_nom_array[-1, :, :]` in lines 165 and 342? Even though the two values are the same, it seems to me like `p.ubi_nom_SS` is cleaner, more conceptually correct.

Reply:
@rickecon This was done to reduce the number of parameters, `ubi_nom_SS` was set from `ubi_nom_array` anyway.  This removes duplication and is consistent without other parameters that vary over the time path are referenced (e.g., see [here](https://github.com/PSLmodels/OG-USA/blob/77e18216ba1e53fe55c87241f5d943f957dad81a/ogusa/SS.py#L193)).